django/core/management/commands/loaddata.py

```diff
@@ -85,6 +92,7 @@ def loaddata(self, fixture_labels):
             None: (open, 'rb'),
             'gz': (gzip.GzipFile, 'rb'),
             'zip': (SingleZipReader, 'r'),
+            'stdin': (lambda x, y: sys.stdin, None),
```

Comment:
They don't matter, don't have significant names. They are just to make function with 2 args, that calling in code later, to return sys.stdin.